### ISSUE TITLE
Xtend Class wizard not showing in navigator's menu

### ISSUE THREAD

**Issue by Hanksha** [eclipse/xtext-xtend#699]
Hi,

For some reason the Xtend Class wizard is not showing in the Project Explorer or Package Explorer menus.
![image](https://user-images.githubusercontent.com/6960133/51647709-d9cc5000-1fb8-11e9-87bb-8b90c802e70f.png)

I'm using the default Java perspective with Eclipse 4.7.3a and Xtend 2.16.0.v20181203-1347

Is it intended or did I misconfigure something? My workaround for now is to use a shortcut for that wizard or use quick access to type the wizard's name.

**Comment by cdietrich** [comment]
i wonder what causes this

**Comment by Hanksha** [comment]
Maybe one of my plugin is conflicting? On the Eclipse 4.9.0 one, where I have less plugin (and fresher install), I can see it. Maybe when I update to Photon it will work again.

**Comment by cdietrich** [comment]
i dont know of anything that would break this in 4.9/2018-09

**Comment by cdietrich** [comment]
would be nice to have it in navigators as well.

**Comment by Hanksha** [comment]
I updated to Eclipse 4.10 2018-12 (with the same Eclipse installation) and the problem persists.

**Comment by cdietrich** [comment]
Do you have something installed that removes stuff from perspectives?

**Comment by Hanksha** [comment]
It could be because of DevStyle, it's the only plugin I have installed that modifies heavily Eclipse.

**Comment by cdietrich** [comment]
could not reproduce with devstyle darkest theme

**Comment by cdietrich** [comment]
<img width="609" alt="bildschirmfoto 2019-02-11 um 10 44 56" src="https://user-images.githubusercontent.com/238736/52555467-20f47680-2dea-11e9-8a3d-531a0b7bf224.png">

**Comment by cdietrich** [comment]
btw i dont see "Java Project" in your screenshot.
=> where and how do your execute what?

**Comment by Hanksha** [comment]
The original screenshot is from the Project Explorer. In the Package Explorer I get this:
![image](https://user-images.githubusercontent.com/6960133/52603077-28765700-2ea0-11e9-9d1d-a6213818c904.png)

The `.project` is:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<projectDescription>
	<name>xtend-examples</name>
	<comment></comment>
	<projects>
	</projects>
	<buildSpec>
		<buildCommand>
			<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>
			<arguments>
			</arguments>
		</buildCommand>
		<buildCommand>
			<name>org.eclipse.jdt.core.javabuilder</name>
			<arguments>
			</arguments>
		</buildCommand>
		<buildCommand>
			<name>org.eclipse.pde.ManifestBuilder</name>
			<arguments>
			</arguments>
		</buildCommand>
		<buildCommand>
			<name>org.eclipse.pde.SchemaBuilder</name>
			<arguments>
			</arguments>
		</buildCommand>
	</buildSpec>
	<natures>
		<nature>org.eclipse.xtext.ui.shared.xtextNature</nature>
		<nature>org.eclipse.pde.PluginNature</nature>
		<nature>org.eclipse.jdt.core.javanature</nature>
	</natures>
</projectDescription>
```

**Comment by Hanksha** [comment]
@cdietrich I did a clean installation of Eclipse 2018-12 and reinstalled my usual plugins. To my surprise if I reuse the same workspace the menu items still don't show up but if I switch to a new workspace they are there.

**Comment by cdietrich** [comment]
i wonder what is in your workspace config that disables the menu entries

**Comment by Hanksha** [comment]
@cdietrich Any idea how those wizards are contributed to the Package Explorer view's popup menu? I was looking for a `org.eclipse.ui.navigator.navigatorContent` extension point with `commonWizard` for those wizards but I can't find it.

**Comment by cdietrich** [comment]
https://github.com/eclipse/xtext-xtend/blob/0d4acbd6573ed05794c28942225af3288bf340d7/org.eclipse.xtend.ide/plugin.xml#L967

**Comment by Hanksha** [comment]
Ah that's right, I wonder why the JDT wizard shortcuts are showing on any navigator but the xtend is specifically not showing in the Project Explorer view (it's showing for the Navigator view)

**Comment by Hanksha** [comment]
I debugged it a bit and turns out that JDT contributes for the Project Explorer and the other navigator like the Package Explorer differently. For the other views it has its own `ActionGroup` (`NewWizardsActionGroup`) that contributes to the menu but for the Project Explorer it's doing it via the `commonWizard` extension point.

So if I add the xtend wizard the same way it now works.

```xml
<extension
         point="org.eclipse.ui.navigator.navigatorContent">
         <commonWizard
               type="new"
               wizardId="org.eclipse.xtend.ide.wizards.NewXtendClassWizard"
               menuGroupId="org.eclipse.jdt.ui.java">
            <enablement></enablement>
         </commonWizard>
</extension>
```

![image](https://user-images.githubusercontent.com/6960133/54402391-c9a63680-4706-11e9-90db-8cbf416614b6.png)

**Comment by kthoms** [comment]
Fixed via 34e5515

**Comment by kthoms** [comment]
Thanks @Hanksha , your help is much appreciated!
